m4_sessions/1708-inning/group_two.md

```diff
@@ -21,16 +21,26 @@ What do we want people attending the session to learn or walk away with at the e
 * Being more actively aware of usability
 
 ### Session Outline:
-Intro to UX/UI (10 minutes)
-- need to come up with what this looks like
-Design Thinking Video (10 minutes or less)
-- need to find one or two great videos
+_We'll use a keynote slide deck to guide the session, which will be shared out at the end of the session_
```

Comment:
Would be great to start a slack channel to share this resource and have people share additional resources too